Content.Shared/Eye/Blinding/Systems/BlurryVisionSystem.cs

```diff
@@ -26,7 +28,17 @@ public void UpdateBlurMagnitude(Entity<BlindableComponent?> ent)
         if (!Resolve(ent.Owner, ref ent.Comp, false))
             return;
 
-        var ev = new GetBlurEvent(ent.Comp.EyeDamage);
+        var maxModificatorStrength = 0;
+
+        if (TryComp<TemporaryBlurryVisionComponent>(ent, out var comp))
+        {
+            if (comp.TemporaryBlurModificators.Count != 0)
+            {
+                maxModificatorStrength = comp.TemporaryBlurModificators.Max(mod => mod.EffectStrength);
+            }
+        }
+
+        var ev = new GetBlurEvent(Math.Max(ent.Comp.EyeDamage, maxModificatorStrength));
```

Comment:
Can this not be moved to an event subscription in an RMC14 specific system to achieve the same thing?